api/candlepin-api-spec.yaml

```diff
@@ -193,6 +193,143 @@ paths:
               schema:
                 $ref: '#/components/schemas/ExceptionMessage'
 
+  /distributor_versions:
+    get:
+      tags:
+        - DistributorVersions
+      description: Retrieves a list of Distributor Versions
+      operationId: getVersions
+      x-java-response-container-type: java.util.stream.Stream
+      security: []
+      parameters:
+        - name: name_search
+          in: query
+          required: false
+          schema:
+            type: string
+        - name: capability
+          in: query
+          required: false
+          schema:
+            type: string
+      responses:
+        200:
+          description: A successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/definitions/DistributorVersionDTO'
+        default:
+          description: An unexpected exception occurred
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+
+    post:
+      tags:
+        - DistributorVersions
+      description: Creates a Distributor Version
+      operationId: create
+      security: []
+      requestBody:
+        description: A new distributor version to be created
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DistributorVersionDTO'
+      responses:
+        200:
+          description: A successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DistributorVersionDTO'
+        409:
+          description: A distributor version already exists
```

Comment:
In this case, the code throws a `BadRequestException`, which is 400, not 409.

Reply:
Done.